packages/web-forms/e2e/README.md

```diff
@@ -35,16 +38,16 @@ e2e/
    ```
 
 2. **Run tests**
-   Execute all E2E tests:
+   Execute all functional E2E tests:
 
    ```bash
-    yarn workspace @getodk/web-forms test:e2e
+    yarn workspace @getodk/web-forms test:e2e:functional:all
    ```
 
    Or run specific tests:
 
    ```bash
-   yarn workspace @getodk/web-forms test:e2e <filepath, e.g. e2e/test-cases/geopoint.test.ts>
+   yarn workspace @getodk/web-forms test:e2e:functional:<browser> <filepath, e.g. e2e/test-cases/geopoint.test.ts>
```

Comment:
Let's add instructions for running the visual tests but explain that they may not pass locally... it's still useful to manually check that you haven't broken anything substantive.